app/models/vha_facility_adapter.rb

```diff
@@ -135,8 +135,10 @@ def service_whitelist
   ).freeze
 
   def self.mh_clinic_phone(attrs)
-    return '' if (attrs['MHClinicPhone']).blank?
-    result = attrs['MHClinicPhone'].to_s
+    val = attrs['MHClinicPhone']
+    val = attrs['MHPhone'] if val.blank?
+    return '' if val.blank?
+    result = val.to_s
```

Comment:
Will either of these two attributes ever be nil? or can they be some combination of nil or ''?

I'm wondering why you don't just do 

```ruby
val = attrs['MHClinicPhone'] || atts['MHVphone']
return '' if val.blank?
```